<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-TFS32G0" failures="1" tests="7" name="org.swiggy.scenario.retryClass" time="0.007" errors="0" timestamp="05 Nov 2022 06:43:46 GMT" skipped="5">
  <testcase classname="org.swiggy.scenario.retryClass" name="test" time="0.000">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at org.swiggy.scenario.retryClass.test(retryClass.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.retryFailed(Invoker.java:839)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1010)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
]]>
    </failure>
  </testcase> <!-- test -->
  <testcase classname="org.swiggy.scenario.retryClass" name="test" time="0.000">
    <skipped/>
  </testcase> <!-- test -->
  <testcase classname="org.swiggy.scenario.retryClass" name="test" time="0.000">
    <skipped/>
  </testcase> <!-- test -->
  <testcase classname="org.swiggy.scenario.retryClass" name="test" time="0.000">
    <skipped/>
  </testcase> <!-- test -->
  <testcase classname="org.swiggy.scenario.retryClass" name="test" time="0.001">
    <skipped/>
  </testcase> <!-- test -->
  <testcase classname="org.swiggy.scenario.retryClass" name="test" time="0.002">
    <skipped/>
  </testcase> <!-- test -->
  <testcase classname="org.swiggy.scenario.retryClass" name="test" time="0.004"/>
</testsuite> <!-- org.swiggy.scenario.retryClass -->
